pkg/cmd/cli/cmd/set/env.go

```diff
@@ -48,6 +48,12 @@ syntax.`
   # Update all containers in all replication controllers in the project to have ENV=prod
   %[1]s env rc --all ENV=prod
 
+  # Import environment from a secret
+  %[1]s env --from-secret=mysecret dc/myapp
```

Comment:
Probably not. I think if you want that, for now, you need it as a file via a volumeMount. Maybe in v2alpha1 we propose extending Secret so you can specify metadata per key (binary, string, etc).